routes/runtime/github.rb

```diff
@@ -78,6 +78,12 @@ class CloverRuntime
           fail CloverError.new(400, "InvalidRequest", "No workflow job data available")
         end
 
+        entry = repository.cache_entries_dataset
+          .exclude(committed_at: nil)
+          .where(key: key, version: version, scope: scope).all
+
+        fail CloverError.new(409, "AlreadyExists", "Scope: #{scope}, Key: #{key}, Version: #{version}") if entry.any?
```

Comment:
Since the current `ubicloud/cache` client also uses this API, I would prefer a more explanatory error message. Something like
```suggestion
        fail CloverError.new(409, "AlreadyExists", "A cache entry for #{scope} scope already exists with #{key} key and #{version} version.") if entry.any?
```